app/views/events/index.html.erb

```diff
@@ -5,6 +5,7 @@
     Click event to add your name to the guest list.
     <thead >
       <tr>
+        <th scope="col">Photo</th>
```

Comment:
Should this appear below name?

Reply:
Tried it, but it looks a little weird.  I think it looks best without the "Photo" header.  Will push and show you what I mean in a few minutes